src/codegen/tests/codegen_error_messages_tests.rs

```diff
@@ -498,3 +498,223 @@ fn pointer_binary_expression_modulo() {
         panic!("expected code-gen error but got none")
     }
 }
+
```

Comment:
if you move this to a new module, you can add #[cfg(features = "verify"] on that module and keep the tests active